### PR TITLE
Sql only

### LINKED ISSUE #19: User Story 10

### PR DESCRIPTION
**Pull request by AuroraTuring**
- User Story 10
- Add an index that views shelters in reverse alphabetical order.
- Test that shelters can be viewed in reverse alphabetical order.

### CODE REVIEW

**Comment by AuroraTuring** [comment]
This branch is still a WIP, please hold off comments and approvals until otherwise specified.

**Comment by AuroraTuring** [comment]
This branch is ready for review!

File: spec/features/admin/shelters/index_spec.rb
Comment (noahdurbin) [inline spec/features/admin/shelters/index_spec.rb:1]: This way of routing for the admin tests makes a lot more sense than the way I started out, Great Work!

File: app/models/shelter.rb [hunk 1]
```diff
@@ -31,4 +31,8 @@ def alphabetical_pets
   def shelter_pets_filtered_by_age(age_filter)
     adoptable_pets.where('age >= ?', age_filter)
   end
+
+  def self.reverse_alphabetical_shelters
+    find_by_sql("SELECT * FROM shelters ORDER BY shelters.name desc")
```
Comment (noahdurbin) [inline app/models/shelter.rb:36]: nice SQL query!

**Review by noahdurbin — APPROVED** [review]

**Pull request merged**